### PR TITLE
DOCS-1131: Include type for ML model package json example

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Comment by zaporter-work** [comment]
🎉 looks good!

File: docs/services/ml/_index.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ Add the `tflite_cpu` ML model object to the services array in your raw JSON conf
     "type": "mlmodel",
     "model": "tflite_cpu",
     "attributes": {
-      "model_path": "${packages.<model-name>}/<model-name>.tflite",
-      "label_path": "${packages.<model-name>}/labels.txt",
+      "model_path": "${packages.ml_model.<model_name>}/<model-name>.tflite",
+      "label_path": "${packages.ml_model.<model_name>}/labels.txt",
```
Comment (sguequierre) [inline docs/services/ml/_index.md:81]: Maybe we could choose a model name like "your_model" for the JSON example here?
```suggestion
      "label_path": "${packages.ml_model.your_model}/labels.txt",
```

**Review by sguequierre — APPROVED** [review]
lgtm one suggestion

File: docs/services/ml/_index.md (hunk 1 above)
Comment (npentrel) [inline docs/services/ml/_index.md:81]:
```suggestion
      "model_path": "${packages.ml_model.my_fruit_model}/my_fruit_model.tflite",
      "label_path": "${packages.ml_model.my_fruit_model}/labels.txt",
```

**Pull request merged**